.github/workflows/release.yml

```diff
@@ -96,11 +108,37 @@ jobs:
         run: |
           make static-dist
 
-  release:
+      - name: Make LICENSE_DEPENDENCIES.md
+        if: inputs.dodistrelease
+        run: |
+          go run github.com/google/go-licenses@v1.6.0 report ./... --ignore github.com/openbao/openbao --template LICENSE_DEPENDENCIES.tpl > LICENSE_DEPENDENCIES.md
+
+      - name: go mod vendor
+        if: inputs.dodistrelease
+        run: |
+          go mod vendor
+
+      - name: Make distribution source tarball
+        if: inputs.dodistrelease
+        id: make-distribution-tarball
+        run: |
+          version="$(git describe --tags --exclude "api/*" --exclude "sdk/*" --abbrev=0 | sed 's/^v//')"
+          echo "version=$version" | tee -a "$GITHUB_OUTPUT"
+          tar --exclude .git --exclude '*.gz' --exclude '*/ui/.yarn/cache' -cJf ../openbao-dist-$version.tar.xz --xform="s,^\.,openbao-dist-$version," .
```

Comment:
If we've already built the UI and it builds into `http/ui`, should we exclude `ui/node_modules` as well?

Reply:
Oh that would be another great savings in the size of the tarball! I will try it.